qubes-rpc/qubes-input-trigger

```diff
@@ -15,10 +16,21 @@ for input_dev in eventFiles:
         if S_ISCHR(os.stat(eventFile).st_mode) != 0:  # is character device?
             udevreturn = subprocess.check_output([
                 "udevadm", "info", "--query=property", "--name=" + eventFile]).decode()
-            if 'ID_INPUT_MOUSE' in udevreturn or \
-                    'ID_INPUT_KEYBOARD' in udevreturn or \
-                    'ID_INPUT_TABLET' in udevreturn or \
-                    'ID_INPUT_TOUCHSCREEN' in udevreturn:
-                subprocess.call([
-                    "sudo", "/bin/udevadm", "trigger",
-                    "--action=add", "--sysname-match=" + input_dev])
+            if '/devices/virtual/' not in udevreturn:
```

Comment:
There should be one more condition some there - to exclude qemu-emulated tablet device.
Bonus points: exclude only the one from stubdomain, but not the one from outer qemu when running nested inside KVM (not sure how to do that, perhaps based on some bus id? usb controller number?)

Formatting-wise, I'd add it below as `if ...:
 continue`.

Reply:
Ok I'll try to have a look.